lib/register_scripts/Classes/DummyRegisterStyles.php

```diff
@@ -33,7 +33,7 @@ class DummyRegisterStyles
     /**
      * Constructs this post
      */
-    public function __construct( $id, $path = NULL, $deps = array(), $version = '0.0.0', $inFooter = true )
+    public function __construct( $id, $path = NULL, $deps = array(), $version = false, $inFooter = false )
```

Comment:
It throw a warning if `DummyRegisterStyles` is called without the `$path`argument